packages/create-app/templates/default-app/packages/app/src/components/search/SearchPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { makeStyles, Theme, Grid, List, Paper } from '@material-ui/core';
+
+import { Content, Header, Page } from '@backstage/core';
+import { CatalogResultListItem } from '@backstage/plugin-catalog';
+import {
+  SearchBar,
+  SearchFilter,
+  SearchResult,
+  DefaultResultListItem,
+} from '@backstage/plugin-search';
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
Bit unfortunate to have the styles here by default tbh, could they be worked into the filter components instead? For the papers I think you should be able to use grid spacing instead

Reply:
Yeah, maybe we can provide variants for these types of components like `<SearchBar variant="paper" />` or something in the future that provide a sane paper/padding for full-page experiences like this one, vs. being more free-form for "any" context.

Reply:
That's probably fine as it is, but I was thinking it might be best to move the padding on the papers to be spacing on the grid instead, just to skip the styling

Reply:
Not sure we can get the desired effect with grid spacing alone. The padding is giving us this:

![spacing-padding-margin](https://user-images.githubusercontent.com/3496491/121522909-e6b93500-c9f5-11eb-83cc-43a49bdd5137.gif)

We can iterate, in any event.